spec/controllers/api/dashboards_get_spec.rb

```diff
@@ -84,6 +84,37 @@ def validate_pagination_link_query_params(link, params)
       expect(data.map { |dashboard| dashboard[:attributes][:"name"] }.uniq).to eq([@dashboard_private_user_1.name])
     end
 
+    it 'with author-title=<string> filter should return only dashboards with "string" in the author title (full match)' do
```

Comment:
I would also add a test to validate sql injection, by setting the author name to something like: `0'; 1=1; drop tables; --`

Reply:
Added a test confirming SQL injections do not happen, thanks!